experimental/packages/opentelemetry-instrumentation-http/src/utils.ts

```diff
@@ -79,6 +80,10 @@ import {
 } from './types';
 import forwardedParse = require('forwarded-parse');
 
+const logger = diag.createComponentLogger({
```

Comment:
Can you pass around the component logger already available in the Instrumentation class? it is weird this is the only place where a logger with same namespace is created as well

Reply:
Yep, can do - addressed in https://github.com/open-telemetry/opentelemetry-js/pull/5091/commits/9f9e97279ebc5dadcb352567b8cbcf724ec81e22 :slightly_smiling_face: